src/commands/apex/run/test.ts

```diff
@@ -201,15 +201,22 @@ export default class Test extends SfCommand<RunCommandResult> {
       skipCodeCoverage: !flags['code-coverage'],
     };
 
-    // cast as TestRunIdResult because we're building an async payload which will return an async result
-    return (await testService.runTestAsynchronous(
-      payload,
-      flags['code-coverage'],
-      flags.wait && flags.wait.minutes > 0 ? false : !(flags.synchronous && !this.jsonEnabled()),
-      undefined,
-      this.cancellationTokenSource.token,
-      flags.wait
-    )) as TestRunIdResult;
+    try {
+      // cast as TestRunIdResult because we're building an async payload which will return an async result
+      return (await testService.runTestAsynchronous(
+        payload,
+        flags['code-coverage'],
+        flags.wait && flags.wait.minutes > 0 ? false : !(flags.synchronous && !this.jsonEnabled()),
+        undefined,
+        this.cancellationTokenSource.token,
+        flags.wait
+      )) as TestRunIdResult;
+    } catch (e) {
+      const error = SfError.wrap(e);
+      error.message = 'There are no apex tests to run in the org';
+      error.actions = ['Ensure Apex Tests exist in the org'];
+      throw error;
```

Comment:
this will swallow all error msgs from the server/library.
can you make it so that if `Always provide a classes, suites, tests, or testLevel property.` is the error, it adds the action suggesting to check there's tests in the org?.